src/treemap/index.js

```diff
@@ -141,8 +141,8 @@ class Treemap extends React.Component {
           .size([innerWidth, innerHeight])
           .padding(padding);
       const structuredInput = hierarchy(data)
-        .sort((a, b) => a.size - b.size)
```

Comment:
it's weird that we weren't getting an error if it didn't exist?

Reply:
It would just end up with undefined-undefined=NaN which I think just leaves things in its original order